fix(curriculum): Minor Reword of Generate Random Whole Numbers with JavaScript (#50101)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript.md

# --description--

> We can generate random decimal numbers with `Math.random()`, but sometimes we need to generate random whole numbers. The following process will give us a random whole number less than `20`:

Reviewer: ```suggestion
You can generate random decimal numbers with `Math.random()`, but sometimes you need to generate random whole numbers. The following process will give you a random whole number less than `20`:
```
